src/EnergyPlus/WindowManager.cc

```diff
@@ -1925,7 +1923,7 @@ namespace Window {
         return num / denom; // dangerous, doesn't check for zero denominator
     }
 
-    Real64 visibleSpectrumAverage(EnergyPlusData &state, gsl::span<Real64 const> p)
+    Real64 visibleSpectrumAverage(EnergyPlusData const &state, gsl::span<Real64 const> p)
```

Comment:
[src/EnergyPlus/WindowManager.cc:1928]:(style),[constParameterReference],Parameter 'state' can be declared as reference to const